files/scripts/selinux/trivalent/trivalent.te

```diff
@@ -78,7 +78,7 @@ allow trivalent_t tmp_t:lnk_file { create unlink };
 allow trivalent_t tmp_t:sock_file { create unlink };
 
 # required for trivalent to be able to detect whether it's the default browser
-allow trivalent_t trivalent_script_exec_t:file { execute getattr read };
+allow trivalent_t trivalent_script_exec_t:file { execute getattr read execute_no_trans ioctl open };
```

Comment:
Probably a better interface to use for this (some kind of pattern). I'll approve for now as breakfix.